src/types/submission.ts

```diff
@@ -83,7 +83,7 @@ export interface IEmailSubmission extends ISubmission {
   getWebhookView(): null
 }
 
-export type IEmailSubmissionSchema = IEmailSubmission & ISubmissionSchema
+export interface IEmailSubmissionSchema extends IEmailSubmission, Document {}
```

Comment:
how does this make a difference?

Reply:
no real difference, could be the same as 

```ts
export type IEmailSubmissionSchema = IEmailSubmission & Document
```

Just needed to ensure no double typing i guess (since `IEmailSubmission` already extends `ISubmission`, but `ISubmissionSchema` extends `ISubmission` and `Document` again)

(But since it was prefixed with `I` I thought to make it an interface (and also all our schemas are interfaces for adding of instance and model methods) for consistency